src/main/java/io/github/homchom/recode/sys/sidedchat/ChatPredicates.java

```diff
@@ -83,7 +84,7 @@ public static Predicate<Component> getMessagePredicate() {
 
     //SUPPORT
     private static final ChatPattern supportChatPattern = new ChatPattern(
-            new ChatPattern.ChatComponent("[SUPPORT] ", TextColor.fromLegacyFormat(ChatFormatting.BLUE),0)
+            new ChatPattern.ChatComponent("[SUPPORT] ", TextColor.fromRgb(new Color(85,127,212).getRGB()),0)
```

Comment:
No reason to use AWT here. You can put in the color directly like `TextColor.fromRgb(0x857fd4)`, or if you want, you can use recode's `RGB` class (either way, merge my commits before committing again).